openedx/adg/lms/applications/models.py

```diff
@@ -16,8 +17,9 @@ class ApplicationHub(TimeStampedModel):
     """
     user = models.OneToOneField(User, related_name='application_hub', on_delete=models.CASCADE, )
     is_prerequisite_courses_passed = models.BooleanField(default=False, )
+    is_written_application_completed = models.BooleanField(default=False, )
     is_application_submitted = models.BooleanField(default=False, )
-    is_assessment_completed = models.BooleanField(default=False, )
+    submission_date = models.DateField(null=True, blank=True, )
```

Comment:
As I have mentioned above, do get latest pull of master and also have a sync with harris bhai.